### PR TITLE
[FAB-17381] Update policy concept for test network

### PR DESCRIPTION
**Pull request by nikhil550**
Signed-off-by: NIKHIL E GUPTA <negupta@us.ibm.com>

#### Type of change

- Documentation update

#### Description

Cherry pick policy concept update to 2.0 release branch

### CODE REVIEW

File: docs/source/policies/policies.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ In this topic, we'll cover:
 * [What is a policy](#what-is-a-policy)
 * [Why are policies needed](#why-are-policies-needed)
 * [How are policies implemented throughout Fabric](#how-are-policies-implemented-throughout-fabric)
-* [Fabric policy domains](#fabric-policy-domains)
+* [Fabric policy domains](#the-fabric-policy-domains)
 * [How do you write a policy in Fabric](#how-do-you-write-a-policy-in-fabric)
 * [Fabric chaincode lifecycle](#fabric-chaincode-lifecyle)
```
Comment (denali49) [inline docs/source/policies/policies.md]: Link does not work because of typo in #fabric-chaincode-lifecyle needs the last 'c' in the word 'lifecycle'

Comment (denali49) [reply docs/source/policies/policies.md]: I can fix or you can.

**Review by denali49 — COMMENTED** [review]
Link does not work because of typo in #fabric-chaincode-lifecyle needs the last 'c' in the word 'lifecycle'

File: docs/source/policies/policies.md (hunk 1 above)
Comment (denali49) [reply docs/source/policies/policies.md]: Technically not part of the original change, but found during review.

Comment (nikhil550) [reply docs/source/policies/policies.md]: going to fix this in another PR. Closing this one.

**Pull request closed without merging**